Packs/Sixgill-DeepInsights/Integrations/Sixgill/Sixgill.yml

```diff
@@ -49,24 +48,24 @@ configuration:
   - Trend Anomaly
   required: false
   type: 15
-description: Fetching Sixgill's alerts as incidents, Sixgill provides alerts that
-  are based on organization assets, enabling you to take proactive steps to eliminate
-  and mitigate your threats.
-display: Sixgill
+- display: Use system proxy settings
+  name: proxy
+  defaultvalue: ""
+  type: 8
+  required: false
+- display: Trust any certificate (not secure)
+  name: insecure
+  defaultvalue: ""
+  type: 8
+  required: false
+description: Define key assets relevant to your brand, industry and geolocation. Obtain bespoke threat intelligence automatically gathered from exclusive deep, dark and surface web sources-delivered straight into the Demisto platform. Detailed actionable insights include the assessment of the event, the relevant intelligence item, and recommendations for action.
+display: Sixgill Deep Insights
 name: Sixgill
 script:
   commands:
-  - arguments:
-    - default: false
```

Comment:
no need to delete the arguments, just `deprecated: true` should be enough

Reply:
done